### PR TITLE
Add sample keys

### PR DESCRIPTION
**Pull request by millicentachieng**

### CODE REVIEW

File: api-reference/beta/api/security-emailthreatsubmissionpolicy-update.md [hunk 1]
```diff
@@ -74,7 +74,8 @@ The following is an example of a request.
 # [HTTP](#tab/http)
 <!-- {
   "blockType": "request",
-  "name": "update_emailthreatsubmissionpolicy"
+  "name": "update_emailthreatsubmissionpolicy",
+  "sampleKeys": ["DefaultReportSubmissionPolicy"]
```
Comment (FaithOmbongi) [inline api-reference/beta/api/security-emailthreatsubmissionpolicy-update.md:78]: It looks like the ID should be a GUID-like String so using `DefaultReportSubmissionPolicy` in the example doesn't accurately represent the API.

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Hi @millicentachieng - I've left comments on two docs, please confirm.

File: api-reference/beta/api/calltranscript-get.md [hunk 2]
```diff
@@ -386,7 +386,8 @@ WEBVTT
 # [HTTP](#tab/http)
 <!-- {
   "blockType": "request",
-  "name": "get_callTranscript_metadatacontent"
+  "name": "get_callTranscript_metadatacontent",
+  "sampleKeys": ["ba321e0d-79ee-478d-8e28-85a19507f456", "MSo1N2Y5ZGFjYy03MWJmLTQ3NDMtYjQxMy01M2EdFGkdRWHJlQ/transcripts/MSMjMCMjNzU3ODc2ZDYtOTcwMi00MDhkLWFkNDItOTE2ZDNmZjkwZGY4"]
```
[the remark this answers is not in the document]
Comment (millicentachieng) [reply api-reference/beta/api/calltranscript-get.md]: Good catch.

File: api-reference/beta/api/security-emailthreatsubmissionpolicy-update.md (hunk 1 above)
Comment (millicentachieng) [reply api-reference/beta/api/security-emailthreatsubmissionpolicy-update.md:78]: @FaithOmbongi If you look at the example in this [file](https://learn.microsoft.com/en-us/graph/api/security-emailthreatsubmissionpolicy-post-emailthreatsubmissionpolicies?view=graph-rest-beta&tabs=http), `DefaultReportSubmissionPolicy` is an id in the response.

**Review by FaithOmbongi — APPROVED** [review]
LGTM!

**Pull request merged**